src/Apps/Artist/Routes/Overview/Components/__tests__/ArtistRecommendations.test.tsx

```diff
@@ -52,22 +52,89 @@ describe("ArtistRecommendations", () => {
     expect(wrapper.find(RecommendedArtist).length).toEqual(1)
   })
 
-  const defaultArtist = {
-    name: "Juan Gris",
-    related: {
-      artists: {
-        edges: [
-          {
-            node: {
-              __id: "QXJ0aXN0OnBhYmxvLXBpY2Fzc28=",
-              __fragments: {
-                RecommendedArtist_artist: {},
-              },
-            },
+  it("Doesn`t render `show more` button when less than three artists", async () => {
+    const wrapper = await getWrapper(defaultArtist)
+
+    expect(wrapper.find("button").length).toEqual(0)
+  })
+
+  it("Renders `show more` button when more than three artists available", async () => {
+    const wrapper = await getWrapper(pagedArtist)
+
+    expect(wrapper.find("button").length).toEqual(1)
+  })
+})
+
+const artistFields = {
+  name: "",
+  formatted_nationality_and_birthday: "",
+  href: "",
+  image: null,
+  artworks_connection: null,
+  is_followed: false,
+  counts: null,
+}
+
+const defaultArtist = {
+  name: "Juan Gris",
+  related: {
+    artists: {
+      pageInfo: {
+        hasNextPage: false,
+        endCursor: "QXJ0aXN0OnBhYmxvLXBpY2Fzc28=",
+      },
+      edges: [
+        {
+          node: {
+            id: "QXJ0aXN0OnBhYmxvLXBpY2Fzc28=",
+            _id: "QXJ0aXN0OnBhYmxvLXBpY2Fzc28=",
```

Comment:
We can remove this `_id` to match the change in the component. The `id` should have the `=` removed so it looks more like the Gravity ID that this field returns.

Reply:
Oh, this is something I forgot to mention in knowledge share. When I moved from using `mockResolvers` to `mockData` in the `<MockRelayRenderer>`, it required hydrating my fixture data with all fields that any component in the tree would need. There is a [downstream component that is referencing `_id`](https://github.com/pepopowitz/reaction/blob/46e6672438af72fd34eecaa94f62073e12891d43/src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx#L109), so it needs to be there. 

This is also why I ended up creating [this object](https://github.com/pepopowitz/reaction/blob/46e6672438af72fd34eecaa94f62073e12891d43/src/Apps/Artist/Routes/Overview/Components/__tests__/ArtistRecommendations.test.tsx#L68) to spread into all of the fixture nodes.